data_collection/api/tracking_api.yaml

```diff
@@ -248,46 +265,71 @@ paths:
             `dimension999`).
 
             It is possible to override action custom dimension by sending ping request with changed value right after tracking request with updated action was sent.
-          type: string
+          schema:
+            type: string
           example: blue
         - name: link
           in: query
           description: External URL opened by the Visitor. It is recommended to set ``url`` parameter to same value.
-          type: string
-          format: URL
+          schema:
+            type: string
+            format: URL
           example: http://help-example.com/how-to/register.html
         - name: download
           in: query
           #TODO: Should we recommend setting url parameter to same value?
           description: URL of downloaded file.
-          type: string
-          format: URL
+          schema:
+            type: string
+            format: URL
           example: http://example.com/files/TermsOfService.pdf
         - name: search
           in: query
           description: |
-            Internal search query. This is used to track what was searched by a visitor in the application. See:
-            [Site search](https://help.piwik.pro/analytics/site-search/).
+            Internal search query. Used for tracking keyword the visitor searched for in the application.
+            See: [Site search](https://help.piwik.pro/support/reports/internal-search-report/).
 
-            Requests using this parameter are excluded from the general page view statistics.
-          type: string
+            Requests with this parameter create Search events.
+          schema:
+            type: string
           example: sherlock holmes
         - name: search_cat
           in: query
+          deprecated: true
           description: |
-            Internal search category. This is used to track if visitor specified a category during a search of the application.
-            See: [Site search](https://help.piwik.pro/analytics/site-search/).
+            Internal search category. Used for tracking category the visitor specified during a search in the application.
+            See: [Site search](https://help.piwik.pro/support/reports/internal-search-report/).
```

Comment:
Do we need to update this link in other versions?

Reply:
What other versions do you have in mind? From anonymized log endpoint?